core/templates/pages/exploration-editor-page/translation-tab/translator-overview/translator-overview.component.ts

```diff
@@ -83,7 +83,7 @@ export class TranslatorOverviewComponent implements OnInit {
   ) { }
 
   canShowTabModeSwitcher(): boolean {
-    return !this.contextService.isExplorationLinkedToStory() && (
+    return this.contextService.isExplorationLinkedToStory() && (
```

Comment:
Weird  -- looks like your PR somehow contains changes from a bunch of other PRs?

Reply:
Reverted the commit and merged again, that fixed it